POOSDKer/Views/LobbyView.swift

```diff
@@ -38,16 +38,22 @@ struct LobbyView: View {
             }
         }
         .onDisappear {
-            if(appState.isHost) {
-                appState.networkingController?.stopHosting();
-            }
-            else {
-                appState.networkingController?.disconnectFromHost()
+            if(!appState.isInGame) {
+                if(appState.isHost) {
+                    
```

Comment:
Why are these not following standard?

Reply:
Do we have a set standard? Would love to have it on record for future PRs